examples/lock-app/nrf5/command_wrapper.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
```

Comment:
Intend to replace this file by chromium's equivlanet, https://source.chromium.org/chromium/chromium/src/+/master:build/gn_run_binary.py